playground/infrastructure/helm-playground/templates/autoscaling-scio.yaml

```diff
@@ -12,15 +12,21 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-apiVersion: autoscaling/v1
+apiVersion: autoscaling/v2
 kind: HorizontalPodAutoscaler
 metadata:
   name: playground-scio
 spec:
-  maxReplicas: 10
-  minReplicas: 2
+  maxReplicas: 4 
+  minReplicas: 2 
   scaleTargetRef:
     apiVersion: apps/v1
     kind: Deployment
     name: playground-scio
-  targetCPUUtilizationPercentage: 90
+  metrics:
+  - type: Resource
+    resource:
+      name: memory
+      target:
+        type: Utilization
+        averageUtilization: 80
```

Comment:
Please add a new line to the end of file

Reply:
Please also remove trailing spaces (yamllint for hints)